cmd/tsbs_generate_queries/uses/devops/common.go

```diff
@@ -22,22 +22,22 @@ const (
 	// DoubleGroupByDuration is the how big the time range for DoubleGroupBy query is
 	DoubleGroupByDuration = 12 * time.Hour
 	// HighCPUDuration is the how big the time range for HighCPU query is
-	HighCPUDuration = 12 * time.Hour
+	HighCPUDuration       = 12 * time.Hour
```

Comment:
was this run through `gofmt`? This is not the way my `gofmt` treats these

Reply:
Ok, agreed - gofmt has different view on this part than I am.
Anyway, I've run all code through `gofmt`, so there should not be any issues like this
Summary: updated